agenda-webapps/src/main/webapp/vue-app/agenda-common/components/pending-date-polls/AgendaPendingInvitationBadge.vue

```diff
@@ -47,8 +47,10 @@ export default {
     },
   },
   created() {
+    this.$root.$on('date-poll-undo-remove', this.countDatePolls);
```

Comment:
you should emit a new event `event-deletion-canceled` when the canceling has **finished**